tests/tests.bats

```diff
@@ -78,7 +78,7 @@ function teardown () {
   cd "${BATS_TEST_DIRNAME}"/.. || false
 
   local ARG_TEST_VERSION
-  local TEST_VERSION="4.3"
+  local TEST_VERSION="4.6"
```

Comment:
This one can be left as 4.3, it is used as an override to test that the version environment variable works when building the container.